net/q/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+q --help | grep 'Show this help message'
```

Comment:
why do not check PKG_VERSION? missing new line at the end of file btw

Reply:
![image](https://github.com/user-attachments/assets/8cb260c0-9515-482d-93fd-205af8ac0486)
Developers use dev in every tag😭